dashboard/app/models/script.rb

```diff
@@ -1439,6 +1444,7 @@ def summarize(include_lessons = true, user = nil, include_bonus_levels = false)
       assigned_section_id: assigned_section_id,
       hasStandards: has_standards_associations?,
       tts: tts?,
+      deprecated: deprecated,
```

Comment:
Should this be `deprecated: deprecated?`?

Reply:
Yes- updated. Thank you!